### PR TITLE
Remove areas from AMR configs and import Oxford

### PR DESCRIPTION
**Pull request by jamesalmond**

### CODE REVIEW

File: lib/tasks/deployment/20190503121932_create_oxford_amr_config.rake [hunk 1]
```diff
@@ -0,0 +1,24 @@
+namespace :after_party do
+  desc 'Deployment task: create_oxford_amr_config'
+  task create_oxford_amr_config: :environment do
+    puts "Running deploy task 'create_oxford_amr_config'"
+
+    reading_fields = '00:30,01:00,01:30,02:00,02:30,03:00,03:30,04:00,04:30,05:00,05:30,06:00,06:30,07:00,07:30,08:00,08:30,09:00,09:30,10:00,10:30,11:00,11:30,12:00,12:30,13:00,13:30,14:00,14:30,15:00,15:30,16:00,16:30,17:00,17:30,18:00,18:30,19:00,19:30,20:00,20:30,21:00,21:30,22:00,22:30,23:00,23:30,24:00'
```
Comment (jamesalmond) [inline lib/tasks/deployment/20190503121932_create_oxford_amr_config.rake]: Not entirely sure whether the readings start at 00:00 or 00:30. Although, we don't have `handle_off_by_one` turned on for any configs so does it even matter? Claudia is going to investigate.

File: lib/tasks/deployment/20190503121932_create_oxford_amr_config.rake [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      description: 'Stark (Oxfordshire etc)',
+      s3_folder: 'stark',
+      s3_archive_folder: 'archive-stark',
+      local_bucket_path: 'tmp/amr_files_bucket/stark',
+      access_type: 'Email',
+      date_format: "%d/%m/%Y",
+      mpan_mprn_field: 'Meter Number',
+      reading_date_field: 'Reading Date',
+      reading_fields: reading_fields.split(','),
+      header_example: "Meter Number,Stark ID,Reading Date,#{reading_fields}"
```
Comment (jamesalmond) [inline lib/tasks/deployment/20190503121932_create_oxford_amr_config.rake]: Not sure what the `Stark IDa field is for. Claudia looking into.

File: lib/tasks/deployment/20190503121932_create_oxford_amr_config.rake [hunk 3]
```diff
@@ -0,0 +1,24 @@
+namespace :after_party do
+  desc 'Deployment task: create_oxford_amr_config'
+  task create_oxford_amr_config: :environment do
+    puts "Running deploy task 'create_oxford_amr_config'"
+
+    reading_fields = '00:30,01:00,01:30,02:00,02:30,03:00,03:30,04:00,04:30,05:00,05:30,06:00,06:30,07:00,07:30,08:00,08:30,09:00,09:30,10:00,10:30,11:00,11:30,12:00,12:30,13:00,13:30,14:00,14:30,15:00,15:30,16:00,16:30,17:00,17:30,18:00,18:30,19:00,19:30,20:00,20:30,21:00,21:30,22:00,22:30,23:00,23:30,24:00'
+    AmrDataFeedConfig.where(
+      description: 'Stark (Oxfordshire etc)',
+      s3_folder: 'stark',
```
Comment (jamesalmond) [inline lib/tasks/deployment/20190503121932_create_oxford_amr_config.rake]: There's nothing that lets us know that it's for a particular area or school in the email so I've scoped these to the data provider. We might need to consider that when we add more schools or areas and get more data from Stark that we might have duplicate filenames.

**Comment by jamesjefferies** [comment]
I’d assume it’s just their internal ID which should uniquely identify a meter in their system? Maybe?!

**Comment by jamesjefferies** [comment]
Usually it is the 00:30 measuring the first 30 minutes of the day, but as we know it isn’t always the case

I thought we did have off by one turned on for, hmm, Frome?

File: lib/tasks/deployment/20190503121932_create_oxford_amr_config.rake (hunk 1 above)
Comment (jamesjefferies) [reply lib/tasks/deployment/20190503121932_create_oxford_amr_config.rake]: Hmm, I thought it was Frome which should have ```handle off by one``` turned on so I'm a bit confused now!

Comment (jamesjefferies) [reply lib/tasks/deployment/20190503121932_create_oxford_amr_config.rake]: Ah, it should be on for Frome historical - no idea why it's off but we're not importing any historic data at the moment either...

File: lib/tasks/deployment/20190503121932_create_oxford_amr_config.rake (hunk 3 above)
Comment (jamesjefferies) [reply lib/tasks/deployment/20190503121932_create_oxford_amr_config.rake]: Yes, good point - something to think about, file format is currently: ```Transition Bath Limited TA Energy Sparks_20190716.csv```

File: lib/tasks/deployment/20190503121932_create_oxford_amr_config.rake (hunk 2 above)
Comment (jamesjefferies) [reply lib/tasks/deployment/20190503121932_create_oxford_amr_config.rake]: I guess it's their unique ID for a supply point? Or is it unique for a school/account?

**Pull request merged**